.github/workflows/pull-request.yaml

```diff
@@ -48,11 +59,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: open-policy-agent/setup-opa@v2
       - name: build wasm bundle
-        run: make client/react/public/opa.wasm
+        run: |
+          curl -Lo eopa https://github.com/StyraInc/enterprise-opa/releases/download/v1.35.0/eopa_Linux_x86_64
```

Comment:
Alternatively, we could have pulled the EOPA caps file and used that with OPA. 🤷 Don't think it matters much which way to go.